### PR TITLE
Update to new login api

### LINKED ISSUE #146: Workspace index improvements
On TPP, there are a lot of workspaces.

On the workspace index, the user is presented with a seemingly unordered list. If they are output checker it is all workspaces.

We should add a sort order, probably alphabetical is simplest, athough sorting by "most recently had updated files" might be nice.

For output-checkers, they should probably just see their explicit workspaces here, rather than all workspaces.  When performing a review, they can navigate to that release request's workspace using the link in the release-request page - they'll still have permission to access it.

### PR DESCRIPTION
**Pull request by bloodearnest**
Use the new API/url, and consume the new metadata it gives us

Fixes #146

### CODE REVIEW

File: airlock/business_logic.py [hunk 1]
```diff
@@ -77,6 +78,12 @@ def __post_init__(self):
         if not self.root().exists():
             raise BusinessLogicLayer.WorkspaceNotFound(self.name)
 
+    def project(self):
+        return self.metadata.get("project", None)
+
+    def __lt__(self, other):
```
Comment (rebkwok) [inline airlock/business_logic.py:84]: I think `@dataclass(order=True)` would mean we don't need this; it compares using the attributes in order, so should still sort properly by name

File: airlock/business_logic.py [hunk 2]
```diff
@@ -77,6 +78,12 @@ def __post_init__(self):
         if not self.root().exists():
             raise BusinessLogicLayer.WorkspaceNotFound(self.name)
 
+    def project(self):
```
Comment (rebkwok) [inline airlock/business_logic.py:81]: Would this be better as a property (or renamed to `get_project`)?

**Review by rebkwok — APPROVED** [review]
Couple of questions, nothing major

File: airlock/business_logic.py (hunk 1 above)
Comment (bloodearnest) [reply airlock/business_logic.py:84]: oh, nice will add

File: airlock/business_logic.py (hunk 2 above)
Comment (bloodearnest) [reply airlock/business_logic.py:81]: That would be my instinct, but I was following @evansd's lead on the original design for PathItems and such. `root` could also be a property

Comment (rebkwok) [reply airlock/business_logic.py:81]: I would probably also make `root` a property, but no very strong feelings either way

**Pull request merged**